specs/redux/locs/actions.spec.js

```diff
@@ -8,31 +8,31 @@ import {store} from '../../init'
 import isEthAddress from '../../../src/utils/isEthAddress'
 import LOCModel from '../../../src/models/LOCModel'
 
-const accounts = UserDAO.web3.eth.accounts
+const account = UserDAO.web3.eth.accounts[0]
 let address
 
 describe('LOCs actions', () => {
-  it('should propose new LOC', (done) => {
-    LOCsManagerDAO.newLOCWatch((locModel) => {
-      expect(locModel.locName).toEqual("Bob's Hard Workers")
-      address = locModel.address
-      done()
-    })
-
+  it('should propose new LOC', done => {
     const data = {
-      locName: "Bob's Hard Workers",
+      locName: '1484554656',
       website: 'www.ru',
       issueLimit: 1000,
       publishedHash: 'QmTeW79w7QQ6Npa3b1d5tANreCDxF2iDaAPsDvW6KtLmfB',
       expDate: 1484554656,
-      account: accounts[0]
+      account
     }
-    store.dispatch(actions.submitLOC(data, () => {
-    }))
+
+    LOCsManagerDAO.newLOCWatch((locModel) => {
+      address = locModel.getAddress()
+      expect(locModel.name()).toEqual('1484554656')
+      done()
+    }, account).then(() =>
+      store.dispatch(actions.submitLOC(data))
```

Comment:
**You've skipped this comment in requested changes. Be more careful!**
You should accept in your action-creators models instead of simple objects